Tasks/AzureRmWebAppDeployment/Tests/L0.ts

```diff
@@ -431,7 +431,8 @@ describe('AzureRmWebAppDeployment Suite', function() {
         assert(tr.stdout.search('JSON - simple string change validated') > 0,'JSON -simple string change validation error' );
         assert(tr.stdout.search('JSON - system variable elimination validated') > 0, 'JSON -system variable elimination validation error');
         assert(tr.stdout.search('JSON - special variables validated') > 0, 'JSON - special variables validation error');
-        assert(tr.stdout.search('JSON - varaibles with dot character validated') > 0, 'JSON varaibles with dot character validated');
+        assert(tr.stdout.search('JSON - varaibles with dot character validated') > 0, 'JSON variables with dot character validation error');
```

Comment:
This assert should be a part of the corresponding test in IISWebAppDeploymentOnMachineGroup Task as well.